index.js

```diff
@@ -252,8 +252,6 @@ async function handleSensorRequest(currentLocationId, radarType) {
   }
   const location = await db.getLocationData(currentLocationId)
 
-  helpers.log(`${currentLocationId}: ${currentState}`)
```

Comment:
New since last time, move this down so that it only prints out when there is a state change.